src/lib/generators/code-snippet-generators.ts

```diff
@@ -21,28 +21,28 @@ export const generateInterfaceDefinition = (
 };
 
 export const generateTypeDefinition = (
-  convertionOptions: MultiFileConvertionOptions | SingleFileConvertionOptions,
+  conversionOptions: MultiFileConvertionOptions | SingleFileConvertionOptions,
```

Comment:
Nice catch - actually we should also rename `MultiFileConvertionOptions` and `SingleFileConvertionOptions`. But thats something for another PR.